modules/gapi/src/streaming/onevpl/source_priv.cpp

```diff
@@ -47,7 +51,7 @@ enum {
 };
 
 GSource::Priv::Priv() :
-    mfx_handle(MFXLoad()),
+//    mfx_handle(MFXLoad()),
```

Comment:
TODO?

Reply:
to do not forget uncomment it after VPL releases their bug fix